rust_src/src/character.rs

```diff
@@ -23,10 +21,7 @@ impl LispObject {
     /// Check if Lisp object is a character or not and return the codepoint
     /// Similar to CHECK_CHARACTER
     pub fn as_character_or_error(self) -> Codepoint {
```

Comment:
Is there any reason to keep this around?

Reply:
I don't think so